recipes/enkits/all/conandata.yml

```diff
@@ -0,0 +1,56 @@
+sources:
+  "1.8":
+    url: "https://github.com/dougbinks/enkiTS/archive/v1.8.zip"
+    sha256: "b96d8d439502d52f9dc419539b12b588df10750f4631ad5518586f7048cdfaef"
+  "1.7":
+    url: "https://github.com/dougbinks/enkiTS/archive/v1.7.zip"
+    sha256: "f74b93c8d64f0f20e9269aab16eaa5747ab54d17595b115c22c5e3ecea043983"
+  "1.6":
+    url: "https://github.com/dougbinks/enkiTS/archive/v1.6.zip"
+    sha256: "e02f9614dc3764f1171434a4b4991505c77d30799ba349f4d8a6697f31f56465"
+  "1.5":
+    url: "https://github.com/dougbinks/enkiTS/archive/v1.5.zip"
+    sha256: "56f74a9ff8ff3160a4f8b2a8da2ac35c8bd46370a1bb480c0ca179ac918fa1ee"
+  "1.4":
+    url: "https://github.com/dougbinks/enkiTS/archive/v1.4.zip"
+    sha256: "35a696db2074284f337e9015976e2a8738aadfb96090dc8116a436246a103024"
+  "1.3":
+    url: "https://github.com/dougbinks/enkiTS/archive/v1.3.zip"
+    sha256: "161efd34e1c5f0572aaf81f69df5f047d6ef06a4dd8c1da75fb5ea32554e0718"
+  "1.2":
+    url: "https://github.com/dougbinks/enkiTS/archive/v1.2.zip"
+    sha256: "72205e1aa0bd47b5710206f31168b846e8a8c1ee906be2b067ec51cdfcdc127d"
+  "1.1":
+    url: "https://github.com/dougbinks/enkiTS/archive/v1.1.zip"
+    sha256: "79df6d5ae4954b23afc5b711af712737af0a75aed42563a3e7494ebd8f4077ba"
+  "1.0":
+    url: "https://github.com/dougbinks/enkiTS/archive/v1.0.zip"
```

Comment:
Do you REALLY need all those versions? We usually start distributing only the latest. Old version are distributed only they are required by other packages.

Reply:
> We usually start distributing only the latest. 
Sorry I didn't know that convention. I thought I might as well add all of them

Reply:
Please, add only the latest version. You can keep an old version if you use for your job.